prime_backup/mcdr/online_player_counter.py

```diff
@@ -103,39 +113,61 @@ def __on_load(self, prev: Any, should_update_from_api: bool):
 		if (prev_lock := getattr(prev, 'data_lock', None)) is not None and type(prev_lock) is type(self.data_lock):
 			with prev_lock:
 				prev_data_is_correct = getattr(prev, 'data_is_correct', False)
-				prev_player_list = getattr(prev, 'player_list', None)
-				if isinstance(prev_player_list, list):
-					prev_player_list = prev_player_list.copy()
+				prev_player_dict: Optional[_PlayerDict] = getattr(prev, 'player_dict', None)
+				if not isinstance(prev_player_dict, dict) or not all(
+						isinstance(record, PlayerRecord)
```

Comment:
This `isinstance` always fails, since the `PlayerRecord` class from the reloaded plugin is different from the `PlayerRecord` class from the old plugin